package.json

```diff
@@ -164,11 +164,18 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after Build",
+      "description": "Run clean command after every successful build.",
```

Comment:
I'm not overly eager on adding a feature that introduces a caveat like this. Instead of trying to "fix" the problem by adding this description, perhaps we should take a step back and look at the broader picture — e.g. would it be better if we changed the default clean pattern to not include result and SyncTeX files? Or perhaps there is some other alternative, that is better than a note in a description (that frankly put, I fear many will overlook/ignore).

**TL; DR** instead of trying to fix the symptoms, fix the cause of the symptoms instead :thought_balloon: